electron/models/Call.ts

```diff
@@ -86,4 +87,25 @@ export default class Call extends BaseEntity {
     )
     @JoinColumn({ name: 'suicide_risk_id' })
     suicideRisk: SuicideRisk;
+
+    @Column('varchar', { name: 'serial_number' })
+    serialNumber: String;
+
+    @Column('bigint', { name: 'date' })
+    Date: Date;
+
+    @Column('bigint', { name: 'time' })
+    Time: Timestamp;
```

Comment:
These two don't actually map 1:1 with the form fields. Instead, the form fields should be used together to generate a unique timestamp which is in the same format as `created_at`.